docs/instrumentations/golang/golang.mdx

```diff
@@ -19,7 +19,10 @@ The following go modules will be auto instrumented by Odigos:
 
 ### HTTP Frameworks
 
-- [`github.com/gin-gonic/gin`](https://pkg.go.dev/github.com/gin-gonic/gin) <sup>*</sup>. HTTP web framework
+<Warning>Please note that in order for the `http.route` parameter to be properly extracted by Go auto instrumentation, 
+you should use Go 1.22 or above.</Warning>
```

Comment:
go 1.22 is not enough to make this attribute appear.

The code also needs to be written in a specific way

Reply:
Amir, I addressed your concerns and updated the doc